tests/core/fs/storage/test_user_storage.py

```diff
@@ -100,10 +103,13 @@ async def test_vacuum(alice_user_storage):
 
 @pytest.mark.trio
 @customize_fixtures(real_data_storage=True)
-async def test_storage_file_tree(tmp_path, alice):
+async def test_storage_file_tree(tmp_path: Path, alice: LocalDevice):
     manifest_sqlite_db = tmp_path / alice.slug / "user_data-v1.sqlite"
 
+    assert (
```

Comment:
this assert seems very paranoid to me: `tmp_path` fixture is guarantee to provide us an empty folder exclusive to our test, so I don't see what we are trying to prevent here

I guess the point is to stress the fact we are testing the creation of the fs file here. In this case I think it would be more understandable to add a comment, e.g.

```python
# Pristine start: DB is not created on FS...
assert not manifest_sqlite_db.exists()
async with UserStorage.run(tmp_path, alice) as aus:
    # ...and now it is !
    assert manifest_sqlite_db.is_file()
```

Reply:
> I guess the point is to stress the fact we are testing the creation of the fs file here

It's the case